packages/react/src/components/scheduler-agenda/scheduler-agenda.tsx

```diff
@@ -0,0 +1,61 @@
+import './scheduler-agenda.scss';
+
+import React, { useCallback } from 'react';
+import { Duration } from 'luxon';
+import List from 'devextreme-react/list';
+import Button from 'devextreme-react/button';
+import { useScreenSize } from '../../utils/media-query';
+
+const getDurationString = (appointment) => {
```

Comment:
Minor: I would move this logic inside the component and wrapped in useMemo, as well as start calculation. But maybe it is not needed here in the demo, since it is more of an optimization. Just note.